<launch>
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="mjpeg" /> <!-- or yuyv -->
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
        <param name="camera_info_url" value=""/>
        <param name="framerate" value="30" />
    </node>
    <!-- Orange Target Detector Node -->
    <node name="color_detector" pkg="color_target_finder" type="color_detector.py" output="screen">
    </node>

    <!-- Motion Commander Node -->
    <node name="motion_commander" pkg="color_target_finder" type="motion_commander.py" output="screen">
    </node>
</launch>
